### PR TITLE
Update multiplayer pong to godot 4 beta 5

### PR DESCRIPTION
**Pull request by scriptsengineer**

### CODE REVIEW

File: networking/multiplayer_pong/logic/paddle.gd [hunk 1]
```diff
@@ -21,7 +21,7 @@ func _process(delta):
 
 		# Using unreliable to make sure position is updated as fast
 		# as possible, even if one of the calls is dropped.
-		rpc_unreliable("set_pos_and_motion", position, _motion)
+		rpc("set_pos_and_motion", position, _motion)
```
Comment (Faless) [inline networking/multiplayer_pong/logic/paddle.gd:24]:
```suggestion
		set_pos_and_motion.rpc(position, _motion)
```

File: networking/multiplayer_pong/logic/paddle.gd [hunk 2]
```diff
@@ -44,6 +44,6 @@ func _hide_you_label():
 
 
 func _on_paddle_area_enter(area):
-	if is_network_master():
-		# Random for new direction generated on each peer.
+	if is_multiplayer_authority():
+		# Random for new direction generated checked each peer.
 		area.rpc("bounce", left, randf())
```
Comment (Faless) [inline networking/multiplayer_pong/logic/paddle.gd:49]:
```suggestion
		area.bounce.rpc(left, randf())
```

**Review by Faless — CHANGES_REQUESTED** [review]
See my comments on the preferred RPC syntax, other RPCs not touched by the code should also be updated to that format.

**Comment by scriptsengineer** [comment]
Update all rpcs with new syntax: [3cab28e](https://github.com/godotengine/godot-demo-projects/pull/792/commits/3cab28e47cf20a2a3d2013cfe7ee59fd7bb5f04b)

**Review by Faless — APPROVED** [review]
Looks good now :+1: , thanks for updating the demo :trophy: .
Note that, before we can merge, you will have to squash you commits together.
See here for more info: https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase

**Comment by scriptsengineer** [comment]
Successfully squashed commits

**Comment by Faless** [comment]
Thanks!

**Pull request merged**